sources/platform/schedules.md

```diff
@@ -6,79 +6,84 @@ category: platform
 slug: /schedules
 ---
 
-# Schedules
-
 **Learn how to automatically start your Actor and task runs and the basics of cron expressions. Set up and manage your schedules from Apify Console or via API.**
 
 ---
 
-[Schedules](https://console.apify.com/schedules) allow you to run your Actors and tasks at specific times. You schedule the run frequency using [cron expressions](#cron-expressions).
+Schedules allow you to run your Actors and tasks at specific times. You schedule the run frequency using [cron expressions](#cron-expressions).
+
+:::note Timezone & Daylight Savings Time
```

Comment:
Question: How does the `note` work? Is that some custom component we have?

Reply:
nope, this is built in into docusaurus as default component, but we can style it if we want to, here is the relevant lint in docusaurus documentation

https://docusaurus.io/docs/2.x/markdown-features/admonitions

Reply:
Nice, didn't know about this.